change/@azure-msal-common-e77b173d-7e21-4295-82f2-5dcc3edf653f.json

```diff
@@ -0,0 +1,7 @@
+{
+  "type": "minor",
```

Comment:
Patch is fine for this change
```suggestion
  "type": "patch",
```

Reply:
I think this should be minor, we are adding something to the public surface.